### ISSUE TITLE
User Bio Has No Length Limit

### ISSUE THREAD

**Issue by stravid87** [globe-and-citizen/Celebrity-Fanalyzer#87]
- User bio should be limited to 150? 100? 200?

**Comment by stravid87** [comment]
Goal: 
1) Solve issue
2) Add appropriate documentation inline
3) Attempt to merge with appropriate testing.

**Comment by stravid87** [comment]
@arnonrdp
This issue is solved I think by just adding the attribute maxlength="150"
```
<q-tab-panel class="q-gutter-y-md" name="profile">
        <q-input v-model="user.displayName" label="Name" />
        <q-input v-model="user.bio" maxlength="150" label="Bio" />
        <h3 class="q-mt-lg text-bold text-h5 text-secondary">MetaData</h3>
        <q-input v-model="user.data1" label="Data 1" />
        <q-input v-model="user.data2" label="Data 2" />
        <q-btn class="full-width" color="primary" label="Save" padding="12px" rounded @click="save()" />
      </q-tab-panel>
```
Doubt it needs its own branch haha. Wanna just add it into any one of the branches you merge?

**Comment by arnonrdp** [comment]
You can commit it directly into the `develop` branch due to simplicity.

On your commit message you can add: `fix #87`
